style.css

```diff
@@ -0,0 +1,173 @@
+html {
```

Comment:
What is this file used for ?
Why is there a `style.css` file AND a `src/style/style.css` file ?